### PR TITLE
Feature/shared files

### PR DESCRIPTION
**Pull request by andriikamaldinov1**

### CODE REVIEW

File: accordion/accordion.component.scss [hunk 1]
```diff
@@ -0,0 +1,217 @@
+@import '../../styles';
```
Comment (xakeppok) [inline accordion/accordion.component.scss:1]: use absolute path

File: accordion/accordion.component.ts [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    ElementRef,
+    EventEmitter,
+    inject,
+    Input,
+    Output,
+    QueryList,
+    ViewChildren,
+} from '@angular/core';
+import { NgClass, NgForOf, NgOptimizedImage, NgStyle } from '@angular/common';
+import { IListItem } from '../../assets/content/content.interfaces';
```
Comment (xakeppok) [inline accordion/accordion.component.ts:13]: Why interface in assets?

File: accordion/accordion.component.ts [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+import { AssetPipe } from '../asset/asset.pipe';
+import { HidePipe } from '../hide/hide.pipe';
+import { VisitBtnComponent } from '../visit-btn/visit-btn.component';
```
Comment (xakeppok) [inline accordion/accordion.component.ts:16]: crate @alias for imports

File: accordion/accordion.component.ts [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+import { ColorPipe } from '../color/color.pipe';
+import { TrackByService } from '../track-by/track-by.service';
+
+@Component({
+    selector: 'ngx-mask-demo-accordion',
```
Comment (xakeppok) [inline accordion/accordion.component.ts:21]: use jsdaddy prefix in submodules

File: chips/chips.component.html [hunk 5]
```diff
@@ -0,0 +1,5 @@
+<div class="chips-block row">
```
Comment (xakeppok) [inline chips/chips.component.html:1]: we have chips on main project please copy their

File: open-source/accordion/accordion.component.html [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+                class="accordion row ai-c bg-black fs-white-14 pointer border-accordion-tb-def"
+                (click)="toggle(i); switchDoc(list.id)"
+                id="{{ list.id }}">
+                <div class="left-side-accordion row ai-c bg-light-grey-accordion">
+                    <img
+                        width="32"
+                        height="32"
+                        [ngSrc]="list.defaultSvg | asset"
+                        class="default-svg"
+                        alt="" />
```
Comment (GlebChiz) [inline open-source/accordion/accordion.component.html:24]: Why is alt empty?

File: open-source/accordion/accordion.component.html [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+                    <img
+                        width="32"
+                        height="32"
+                        [ngSrc]="list.activeSvg | asset"
+                        class="active-svg"
+                        alt="" />
```
Comment (GlebChiz) [inline open-source/accordion/accordion.component.html:30]: Why is alt empty?

File: open-source/accordion/accordion.component.html [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+<div class="drawer-container column bg-black" [ngClass]="showNav | hide">
+    <div class="drawer bg-black">
+        <div
+            *ngFor="let list of lists; let i = index; trackBy: trackByPath"
+            class="non-active accordion-block"
+            #accordion>
+            <div
+                class="accordion row ai-c bg-black fs-white-14 pointer border-accordion-tb-def"
+                (click)="toggle(i); switchDoc(list.id)"
+                id="{{ list.id }}">
```
Comment (GlebChiz) [inline open-source/accordion/accordion.component.html:17]: Usually we use such way [id]="list.id"

**Review by GlebChiz — APPROVED** [review]

File: open-source/chip/chip.component.ts [hunk 9]
```diff
@@ -0,0 +1,20 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+    selector: 'jsdaddy-open-source-chip[chip]',
```
Comment (xakeppok) [inline open-source/chip/chip.component.ts:4]: it's not only open source

**Pull request merged**